frontend/src/scenes/session-recordings/player/PlayerMeta.tsx

```diff
@@ -250,3 +255,68 @@ export function PlayerMeta(): JSX.Element {
         </DraggableToNotebook>
     )
 }
+
+const MenuActions = (): JSX.Element => {
```

Comment:
<img width="733" alt="Screenshot 2024-04-17 at 19 22 47" src="https://github.com/PostHog/posthog/assets/984817/9943534e-3f70-4d7d-9970-a826003d8847">

that positioning makes it feel associated with the person instead of the recording... 

it could go to the right of pin? which opens the road to that menu of actions becoming flexible and pushing more into the kebab menu on smaller screens...